articles/azure-monitor/app/sharepoint.md

```diff
@@ -11,7 +11,7 @@ ms.date: 09/08/2020
 Azure Application Insights monitors the availability, performance and usage of your apps. Here you'll learn how to set it up for a SharePoint site.
 
 > [!NOTE]
-> Due to security concerns, you can't directly add the script that's described in this article to your webpages in the SharePoint modern UX. As an alternative, you can use [SharePoint Framework (SPFx)](/sharepoint/dev/spfx/extensions/overview-extensions) to build a custom extension that you can use to install Application Insights on your SharePoint sites. To learn more, see [How to create a SPFx extension solution with AppInsights installed from scratch](https://github.com/microsoft/ApplicationInsights-JS/tree/master/SPO#how-to-create-a-spfx-extension-solution-with-appinsights-installed-from-scratch) or [view the sample](https://github.com/microsoft/ApplicationInsights-JS/tree/master/SPO/AppInsightsExtensionSolutionSample-NPM). 
+> Due to security concerns, you can't directly add the script that's described in this article to your webpages in the SharePoint modern UX. As an alternative, you can use [SharePoint Framework (SPFx)](/sharepoint/dev/spfx/extensions/overview-extensions) to build a custom extension that you can use to install Application Insights on your SharePoint sites. To learn more, view the [NPM based sample for SharePoint](https://github.com/pnp/sp-dev-fx-extensions/tree/master/samples/js-application-appinsights-advanced) that installs Application Insights via [NPM](./javascript.md#npm-based-setup) or the [JavaScript snippet based sample for SharePoint](https://github.com/pnp/sp-dev-fx-extensions/tree/master/samples/js-application-appinsights) that installs the [Application Insights JavaScript snippet](https://github.com/microsoft/ApplicationInsights-JS/tree/master/AISKU/snippet#how-to-inject-javascript-with-sharepoint-framework-extensions-through-snippet-setup).
```

Comment:
Thanks @DawgFan I add a hyper link for NPM pointing to this doc, and a hyper link for Snippet pointing to AppInsights github doc, because the snippet for SP is slightly different from general cases.
@mrbullwinkle I updated the Snippet doc, you pointed it to this doc, but the snippet for SP is slightly different from general cases. We had it documented in our github doc [here](https://github.com/microsoft/ApplicationInsights-JS/tree/master/AISKU/snippet#how-to-inject-javascript-with-sharepoint-framework-extensions-through-snippet-setup).